docs/get_started/get_started_demos.md

```diff
@@ -561,27 +429,27 @@ To run the **Image Classification** code sample with an input image using the IR
 
 @endsphinxdirective
 
-The following commands run the Image Classification Code Sample using the `car.png` file from the `demo` directory as an input image, the model in IR format from the `ir` directory, and on different hardware devices:
+The following commands run the Image Classification Code Sample using the [`car.png`](https://storage.openvinotoolkit.org/data/test_data/images/car.png) file as an input image, the model in IR format from the `ir` directory, and on different hardware devices:
```

Comment:
it's not rendered well for me..

Reply:
replace squeezenet1.1 with googlenet-v1. Use https://storage.openvinotoolkit.org/data/test_data/images/224x224/dog.bmp for classification